### PR TITLE
BREAKING CHANGE(web): Switch Button to v3 design tokens and remove inverted variant #DS-1445

### PR DESCRIPTION
**Pull request by crishpeen**

### CODE REVIEW

**Review by literat — APPROVED** [review]
LGTM 🤷

File: packages/web/src/scss/components/Button/_Button.scss [hunk 1]
```diff
@@ -2,30 +2,43 @@
 
 @use 'sass:map';
 @use 'theme';
-@use 'tools';
+@use '../../settings/globals';
 @use '../../settings/cursors';
```
Comment (adamkudrna) [inline packages/web/src/scss/components/Button/_Button.scss:6]: 🔤

File: packages/web/src/scss/components/Alert/_theme.scss [hunk 2]
```diff
@@ -1,31 +1,23 @@
...
 
-$gap: tokens.$space-600;
-$padding: tokens.$space-600;
-$border-radius: tokens.$radius-200;
-$border-style: tokens.$border-style-100;
-$border-width: tokens.$border-width-100;
+$gap: global-tokens.$space-700;
+$padding: global-tokens.$space-700;
+$border-radius: global-tokens.$radius-300;
+$border-style: solid;
```
Comment (adamkudrna) [inline packages/web/src/scss/components/Alert/_theme.scss:10]: Shall we still keep it in the theme?

File: tests/e2e/demo-components-compare.spec.ts-snapshots/unstable-truncate-chromium-linux.png
Comment (adamkudrna) [inline tests/e2e/demo-components-compare.spec.ts-snapshots/unstable-truncate-chromium-linux.png:1]: This should be commented out again.

**Review by adamkudrna — APPROVED** [review]

File: packages/web/src/scss/components/Alert/_theme.scss (hunk 2 above)
Comment (crishpeen) [reply packages/web/src/scss/components/Alert/_theme.scss:10]: Currently, I do this everywhere. TBH I don't care where we set it. I will keep it here as I do it elsewhere now, but we can move it in the future without problems.

**Pull request merged**